docs/reference/api/connectivity/lorawan/lorawan.md

```diff
@@ -0,0 +1,70 @@
+<h2 id="lorawan-api">LoRaWAN</h2>
+
+LoRaWAN is a technology designed for low-power battery powered devices. These devices operate in an unlicensed spectrum, creating high desnity wide-area networks.
+
+Arm Mbed OS provides a native network stack for LoRaWAN, which can run on any Mbed Enabled device with a LoRa radio onboard. 
+
+The [LoRaWANInterface](https://github.com/ARMmbed/mbed-os/blob/feature-lorawan/features/lorawan/LoRaWANInterface.h) provides a C++ API for connecting to the internet over a LoRa network.
```

Comment:
We prefer not to link to GitHub. Is there a Doxygen page we could link to, instead?

Reply:
@AnttiKauppila This relates to https://github.com/ARMmbed/Handbook/pull/414#discussion_r171809779. Please clarify the reason to Amanda.

Reply:
@AnotherButler Please check this https://github.com/ARMmbed/Handbook/pull/414#discussion_r171816469

Reply:
We should use real doxygen tag here, but can we change this after release Amanda when we know the real path? Or is there already some doxygen generated for the release?

Reply:
This is valid link currently:
https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_lo_ra_w_a_n_interface.html